bob/src/api/mod.rs

```diff
@@ -313,31 +319,69 @@ async fn status<A: Authenticator>(bob: Extension<BobServer<A>>) -> Json<Node> {
 async fn get_space_info<A: Authenticator>(
     bob: Extension<BobServer<A>>,
 ) -> Result<Json<SpaceInfo>, StatusExt> {
+    let mut disk_metrics = bob.grinder().hw_counter().update_space_metrics();
     let DiskSpaceMetrics {
         total_space,
         used_space,
         free_space,
-    } = bob.grinder().hw_counter().update_space_metrics();
+    } = disk_metrics.values().sum();
```

Comment:
Same here: this sum is incorrect, because multiple bob disks can be mapped to a single physical disk

Reply:
see comment above